theme/helpers/_fonts.scss

```diff
@@ -20,11 +25,11 @@ $ck-font-face: Helvetica, Arial, Tahoma, Verdana, Sans-Serif !default;
  * @see ck-font-size
  */
 $ck-font-sizes: (
-	-2: 0.7rem,
-	-1: 0.75rem,
-	 0: 1rem,
-	 1: 1.4rem,
-	 2: 1.8rem
+	-2: 0.7em,
```

Comment:
Why rem->em?

Reply:
Because if `rem` is in use, the UI elements won't react to font changes of their parents. E.g. the icons will not scale when the button font size changes. It will remain relative to the `<body>` font size.